src/constants/index.ts

```diff
@@ -86,3 +92,13 @@ export const STAGE = process.env.STAGE || 'dev';
 export const RAW_USER_DATA_BUCKET_NAME = `fdbt-raw-user-data-${STAGE}`;
 export const USER_DATA_BUCKET_NAME = `fdbt-user-data-${STAGE}`;
 export const MATCHING_DATA_BUCKET_NAME = `fdbt-matching-data-${STAGE}`;
+
+export const passengerTypesList: PassengerAttributes[] = [
```

Comment:
I'd probably keep this in snake case and all caps to be consistent with the other constants